vxc.go

```diff
@@ -286,6 +290,26 @@ func (svc *VXCServiceOp) UpdateVXC(ctx context.Context, id string, req *UpdateVX
 		update.BEndInnerVLAN = req.BEndInnerVLAN
 	}
 
+	if req.AEndPartnerConfig != nil {
+		partnerConfig := req.AEndPartnerConfig
+		switch partnerConfig.(type) {
+		case *VXCOrderVrouterPartnerConfig:
+			update.BEndPartnerConfig = partnerConfig
+		default:
+			return nil, errors.New("a end partner config type not supported for VXC update")
```

Comment:
I would mention here, and on the B end, that the only supported end config is VRouter.